Build/libHttpClient.GDK/libHttpClient.GDK.def

```diff
@@ -16,6 +16,7 @@ EXPORTS
     HCHttpCallGetRequestUrl
     HCHttpCallPerformAsync
     HCHttpCallRequestEnableGzipCompression
+    HCHttpCallSetCompressResponse
```

Comment:
Add new public APIs to the end of the .def file.  This allows hot swapping a new dll into a game built with an old import lib without things breaking.